### PR TITLE
HDDS-7199. Implement new mix workload Read/Write Freon command which meets specific test requirements

### PR DESCRIPTION
**Pull request by DaveTeng0**
## What changes were proposed in this pull request?
In Cisco/Intel cluster, measure r/w performance when there is a very large of amount of metadata in rocksDB, & different amount of working sets whose size are larger than cache available.

Pure read
Pure write
Mixed workload, Read + Write

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-7199

## How was this patch tested?
Robot tests, manual tests.

### CODE REVIEW

**Comment by DaveTeng0** [comment]
cc. @kerneltime

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RangeKeysGenerator.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import java.util.HashMap;
+
+import static org.apache.hadoop.ozone.freon.KeyGeneratorUtil.PURE_INDEX;
+import static org.apache.hadoop.ozone.freon.KeyGeneratorUtil.MD5;
+import static org.apache.hadoop.ozone.freon.KeyGeneratorUtil.FILE_DIR_SEPARATOR;
+
+/**
+ * Ozone range keys generator for performance test.
+ */
+@CommandLine.Command(name = "ork",
```
Comment (adoroszlai) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RangeKeysGenerator.java:43]: Will we have elves, too? ;)

**Review by adoroszlai — APPROVED** [review]
Thanks @DaveTeng0 for fixing checkstyle and findbugs problems.  Approving based on #3754.

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RangeKeysGenerator.java (hunk 1 above)
Comment (DaveTeng0) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RangeKeysGenerator.java:43]: Yes!

**Pull request merged**